bluebrain/repo-bluebrain/packages/py-visual-cortex-analysis/package.py

```diff
@@ -12,7 +12,7 @@ class PyVisualCortexAnalysis(PythonPackage):
     homepage = "https://bbpgitlab.epfl.ch/circuits/proj120/visual-cortex-analysis"
     git = "ssh://git@bbpgitlab.epfl.ch/circuits/proj120/visual-cortex-analysis.git"
 
-    version("0.1.0.dev0", tag="visual-cortex-analysis-v0.1.0.dev0")
+    version("0.2", tag="visual-cortex-analysis-v0.2")
```

Comment:
Your tagger did not run: https://bbpgitlab.epfl.ch/circuits/proj120/visual-cortex-analysis/-/pipelines/192871

Spack fails because it can't find this tag.